plugins/org.jkiss.dbeaver.ext.dm/src/org/jkiss/dbeaver/ext/dm/model/DMSchema.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.dm.model;
+
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.generic.model.GenericCatalog;
+import org.jkiss.dbeaver.ext.generic.model.GenericDataSource;
+import org.jkiss.dbeaver.ext.generic.model.GenericSchema;
+import org.jkiss.dbeaver.model.DBPEvaluationContext;
+import org.jkiss.dbeaver.model.DBPObjectStatisticsCollector;
+import org.jkiss.dbeaver.model.DBPQualifiedObject;
+import org.jkiss.dbeaver.model.DBUtils;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+
+/**
+ * @author Shengkai Bai
+ */
+public class DMSchema extends GenericSchema implements DBPQualifiedObject, DBPObjectStatisticsCollector {
+    private String schemaName;
+    private boolean hasStatistics;
+
+    public DMSchema(GenericDataSource dataSource, GenericCatalog catalog, String schemaName) {
+        super(dataSource, catalog, schemaName);
+        this.schemaName = schemaName;
+    }
+
+    @Override
+    public boolean isStatisticsCollected() {
+        return hasStatistics;
```

Comment:
You need to read statistics or do not use DBPObjectStatisticsCollector at all.
And without statistics, I do not see any reason for the DMSchema class to exist.